src/initializers/auth.js

```diff
@@ -0,0 +1,54 @@
+const LocalStorageAvailable = require('utils/check-local-storage');
+const AuthActions = require('actions/auth');
+const AuthSelectors = require('selectors/auth');
+
+module.exports = (store) => {
+
+    if (!LocalStorageAvailable()) {
+
+        // This forces strange-auth auth status to be set
```

Comment:
What "auth status" gets set by this?  Reviewing [strange auth docs](https://github.com/BigRoomStudios/strange-auth#usage), are you referring to the actual `status` piece of state, or does this get `token: false` into `credentials`?  I have assumptions, but haven't totally tracked down what the purpose of this is for.  It feels like this is something the default state could cover?  Maybe passed in as [initial state in the auth reducer](https://github.com/BigRoomStudios/strange-auth/blob/master/src/reducer.js#L6)?

Reply:
Yep! I mean the actual `status` piece of state. So you're suggesting that we set initial status state in the auth reducer? But how would we know what that state should look like? 

The way Devin explained it to me is that there are kind of three things we might care about: if a user is authenticated, if a user is not authenticated, or whether we even care about authentication at all. The initial state might look different depending on project/user needs, so wouldn't we want to manage that logic in the initializer instead of defaulting to an initial state in the reducer? I could be 100% wrong about all this tho.